src/os/bluestore/BlueFS.cc

```diff
@@ -1173,12 +1173,15 @@ int BlueFS::fsck()
   return 0;
 }
 
-int BlueFS::_write_super(int dev, bool skip_set_wal_v2)
+int BlueFS::_write_super(int dev, bool force_wal_v1)
```

Comment:
Sorry for not proposing that before but providing exact WAL version (as a "force" mode) or 0 overwise would look even better.